### ISSUE TITLE
no updates for 4 years. is this a dead project?

### ISSUE THREAD

**Issue by thefunkygibbon** [Monitorr/Monitorr#239]
only just found this exists and it looks great. but i hate using dead products.  is this dead?  are there any active forks?

**Comment by skapytek** [comment]
dead

**Comment by robson90** [comment]
@skapytek do you know any alternative ?

**Comment by rob1998** [comment]
Yeah I'm afraid it's dead, I speak for myself when I say I'm no longer developing this.
As for alternatives, I'm using https://github.com/louislam/uptime-kuma atm, which ticked all my requirements, but this depends on what you're looking for ofc.
